modules/ROOT/pages/upgrade-policies-proxies.adoc

```diff
@@ -0,0 +1,128 @@
+= Upgrade Java for Policies and API Proxies 
+
+Upgrade your policies and API proxies to be compatible with Java 17.
+
+[[upgrading-automated-policies]]
+== Upgrading Automated Policies
+
+Upgrade your automated policies (included and custom) to Java 17.
+
+[[upgrading-included-automated-policies]]
+=== Upgrading Included Automated Policies
+
+To upgrade automated xref:mule-gateway::policies-included-directory.adoc[MuleSoft included policies] to Java 17:
+
+. In API Manager, go to *Automated Policies*.
+. In *Policy version*, select the latest version of the included policy.
+. In *Rule of application*:
+  * To apply included policies to all Mule runtimes, select *All runtimes* and click *Apply*.
+  * To apply the included policies to Flex Gateways only, select *Flex Gateways only* and click *Apply*.
+  * To apply the included policies to Mule Gateways only:
```

Comment:
Does the indentation work? Never tried this, please confirm in a local build. Otherwise, use **, but it this works, great!